pygaggle/data/msmarco.py

```diff
@@ -79,11 +80,14 @@ def load_queries(cls,
     def from_folder(cls,
                     folder: str,
                     split: str = 'dev',
-                    is_duo: bool = False) -> 'MsMarcoDataset':
+                    is_duo: bool = False,
+                    run_path: Path = '.') -> 'MsMarcoDataset':
         run_mono = "mono." if is_duo else ""
         query_path = os.path.join(folder, f"queries.{split}.small.tsv")
         qrels_path = os.path.join(folder, f"qrels.{split}.small.tsv")
-        run_path = os.path.join(folder, f"run.{run_mono}{split}.small.tsv")
+        if not os.path.isfile(run_path):
+            run_path = os.path.join(folder, f"run.{run_mono}{split}.small.tsv")
+        print(run_path)
```

Comment:
Can this print statement add a little more context, maybe two depending on if the file exists? Also it should be using logging I think?